module/spl/spl-condvar.c

```diff
@@ -231,7 +231,7 @@ EXPORT_SYMBOL(__cv_timedwait_sig);
  * 'expire_time' argument is an absolute clock time in nanoseconds.
  * Return value is time left (expire_time - now) or -1 if timeout occurred.
  */
-static clock_t
+static hrtime_t
 __cv_timedwait_hires(kcondvar_t *cvp, kmutex_t *mp, hrtime_t expire_time,
```

Comment:
So at the moment it's harmless but let's preserve the existing upstream prototype, and even better for the return value.